lib/Teams.js

```diff
@@ -11,14 +11,17 @@ function TeamsBot(configuration) {
     var controller = Botkit(configuration || {});
 
     controller.api = TeamsAPI(configuration || {});
-
-    controller.api.getToken(function(err) {
+    function tokenHandler(err) {
         if (err) {
             // this is a fatal error - could not create a Teams API client
             throw new Error(err);
         }
-    });
-
+        //use the expires_in value, convert to ms and remove 10 mins in ms
+        let tokenExpiresIn = 1000 * 60 * configuration.token_expires_in;
```

Comment:
In case configuration.token_expires_in is not set (undefined) , we should have default value as 60 minutes?
`let tokenExpiresIn = 1000 * 60 * (configuration.token_expires_in || 60);`

Reply:
How about setting `configuration.token_expires_in` in TeamsAPI.js -https://github.com/howdyai/botkit/blob/master/lib/TeamsAPI.js#L172

`configuration.token_expires_in = json.expires_in`

Reply:
you know what @narenderpal I forgot to commit the token line hahaha, i'd done exactly that and forgot the commit hahaha

Reply:
If I wrote the tests i'd have caught that too ;) i'll work on adding test coverage soon.